### ISSUE TITLE
Support for Gigabyte X99-UD5 WIFI-CF

### ISSUE THREAD

**Issue by downloadingvirus** [openhardwaremonitor/openhardwaremonitor#771]
Hi any chance you look at support for Gigabyte X99-UD5 WIFI-CF mainboards?  There are no sensors at all from mainboard.

Thanks.

[OpenHardwareMonitor.Report.txt](https://github.com/openhardwaremonitor/openhardwaremonitor/files/85643/OpenHardwareMonitor.Report.txt)

**Comment by ElPumpo** [comment]
does http://openhardwaremonitor.org/files/openhardwaremonitor-v0.7.1.2-alpha.zip work?

**Comment by downloadingvirus** [comment]
Yes I upgraded to the alpha and it has recognized the chip as ITE IT8620E.  The two on board temperature sensors are working,  However the board has 5 pwm fan headers but only shows RPM for 2 and it has picked up 3 fan controls but is not showing any data only a "-"

**Comment by ElPumpo** [comment]
Yeah, v0.7.1.2 is unable to control the fans, see #756 for more info.
However, I am unsure about all fans being detected.
